lib/strategy.js

```diff
@@ -70,24 +72,25 @@ Strategy.prototype.authenticate = function(req, options) {
   options = options || {};
   var username = lookup(req.body, this._usernameField) || lookup(req.query, this._usernameField);
   var password = lookup(req.body, this._passwordField) || lookup(req.query, this._passwordField);
-  
+  var totp = lookup(req.body, this._totpField) || lookup(req.query, this._totpField);
+
   if (!username || !password) {
     return this.fail({ message: options.badRequestMessage || 'Missing credentials' }, 400);
   }
```

Comment:
This strategy already complains on missing username or password.

But we _don't_ add a similar failure for missing totp here, because it may legitimately not be required depending on the upstream state of `ClientApplication#enforces_otp?` and `User#second_factor_enabled?`